docs/user-guide/README.md

```diff
@@ -0,0 +1,213 @@
+# User Guide
+
+## :cd: Installation
+
+Use [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/) to install.
+
+```bash
+npm install --save-dev eslint eslint-plugin-vue@next
+```
+```bash
+yarn add eslint eslint-plugin-vue@next --dev
+```
+
+::: tip Requirements
+- ESLint v5.0.0 or later
+- Node.js v6.5.0 or later
+:::
+
+## :book: Usage
+
+### Configuration
+
+Create `.eslintrc.*` file to configure rules. See also: [http://eslint.org/docs/user-guide/configuring](http://eslint.org/docs/user-guide/configuring).
+
+Example **.eslintrc.js**:
+
+```js
+module.exports = {
+  extends: [
+    // add more generic rulesets here, such as:
+    // 'eslint:recommended',
+    'plugin:vue/recommended'
+  ],
+  rules: {
+    // override/add rules settings here, such as:
+    // 'vue/no-unused-vars': 'error'
+  }
+}
+```
+
+See [the rule list](../rules/README.md) to get the `extends` &amp; `rules` that this plugin provides.
+
+#### Use together with custom parsers
+
+If you want to use custom parsers such as [babel-eslint](https://www.npmjs.com/package/babel-eslint) or [typescript-eslint-parser](https://www.npmjs.com/package/typescript-eslint-parser), you have to use `parserOptions.parser` option instead of `parser` option. Because this plugin requires [vue-eslint-parser](https://www.npmjs.com/package/vue-eslint-parser) to parse `.vue` files, so this plugin doesn't work if you overwrote `parser` option.
+
+```diff
+- "parser": "babel-eslint",
+  "parserOptions": {
++     "parser": "babel-eslint",
+      "sourceType": "module"
+  }
+```
+
+### Command
+
+You have to include the `.vue` extension using [the `--ext` option](https://eslint.org/docs/user-guide/configuring#specifying-file-extensions-to-lint) or a glob pattern because ESLint targets only `.js` files by default.
+
+Examples:
+
+```bash
+eslint --ext .js,.vue src
+eslint "src/**/*.{js,vue}"
+```
+
+### Editor integrations
```

Comment:
I think this would be nice as a separate section. just as `Usage`. It will be easier to navigate to it using side nav.